Makefile

```diff
@@ -20,7 +21,7 @@ SHELL = /bin/bash
 .SHELLFLAGS = -e -o pipefail -c
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS = "crd:crdVersions=v1,maxDescLen=220"
+CRD_OPTIONS = "crd:crdVersions=v1,maxDescLen=0"
```

Comment:
I had to remove the description to get around the size limit.

```
Error from server (Invalid): error when creating "STDIN": CustomResourceDefinition.apiextensions.k8s.io "mysqlclusters.moco.cybozu.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

Reply:
The size limit could have been circumvented with `maxDescLen=30`, but 30 characters are too small, so I set it to 0.

Reply:
That's unavoidable but a pity.
Hopefully, we may replace corev1.PodSpec/ServiceSpec with ApplyConfigurations in k8s.io/client-go
so that we can greatly reduce the volume of descriptions.

Reply:
I will make the changes to ApplyConfigurations in another Pull Request.